### ISSUE TITLE
Imports are pushed to the top (above copy right statement)

### ISSUE THREAD

**Issue by sandy081** [Microsoft/vscode#48536]
Testing #48303

- Enable organise imports setting

```json
"editor.codeActionsOnSave": {
    "source.organizeImports": true
}
```

- Open a TS file and do save

🐛 Imports are moved to the top

![kapture 2018-04-24 at 13 55 36](https://user-images.githubusercontent.com/10746682/39185380-4c904724-47c7-11e8-9a80-5de479fc0915.gif)

**Comment by mjbvz** [comment]
Are you using TS 2.8.3?

**Comment by sandy081** [comment]
No I am in `2.8.1`

**Comment by mjbvz** [comment]
This should be fixed if you use TS 2.8.3 in the workspace. Let me know if you still see the problem
